lib/perl/Genome/Model/ClinSeq/Command/IdentifyLoh.pm

```diff
@@ -255,11 +262,12 @@ sub filter_loh {
 #Get rid of intermediate files
 sub cleanup {
     my $self = shift;
+    my $outdir = shift;
     my @patterns = qw(*.Somatic* *.readcounts *.hc *.LOH
```

Comment:
Do we actually need all of these files in the final result under non debugging conditions?